File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n=len(nums)
        ans=[0]*(2*n)
        x=0
        for i in range(2*n):
            ans[x] = nums[i%n]
            x+=1
        return ans

File: Data Structures & Algorithms/first-missing-positive/submission-0.cpp
class Solution {
public:
    int firstMissingPositive(vector<int>& nums) {

        unordered_set<int> numidx;
        int n = nums.size();

        for( int i = 0; i < n; i++)
        {

             numidx.insert(nums[i]);
     }

     for( int i = 1; i <= n + 1; i++) {
    
             

        if(!numidx.count(i))return i ;
        
     }
     return 0;
     }


};
